libs/medic-users-meta/migrations/202304140052.do.telemetry_devices.sql

```diff
@@ -0,0 +1,53 @@
+DROP MATERIALIZED VIEW IF EXISTS useview_telemetry_devices;
+
+CREATE MATERIALIZED VIEW public.useview_telemetry_devices
+TABLESPACE pg_default AS 
+
+SELECT
```

Comment:
I know we don't have `metrics` in the devices stuff but I'd probably still try to make the columns align as closely as possible with the other query, at a minimum, putting `telemetry_doc_id` first.

Reply:
[here](https://github.com/medic/cht-couch2pg/pull/130#discussion_r1166860748) you say you like to put the unique identifier first. In this case, the unique identifier is `{ device_id, username }`. Do you have a preference as-to which of these is more important?

Reply:
Yeah, but since the unique identifier of the couch doc is the `telemetry_doc_id`, that's the field I like to put first. So to me, it satisfies both of my preferences (unique first AND alignment).